agent_server.py

```
@@ -5,6 +5,7 @@
 from datetime import datetime
 import argparse
 from injective_functions.factory import InjectiveClientFactory
+from injective_functions.snapshot import Snapshot
```

Ruff (0.8.0)

`injective_functions.snapshot.Snapshot` imported but unused

Remove unused import: `injective_functions.snapshot.Snapshot`

(F401)